python/cudf/cudf/tests/test_text.py

```diff
@@ -0,0 +1,162 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+
+import pytest
+from pandas.util.testing import assert_series_equal
+
+import cudf
+
+
+def test_tokenize():
+    strings = cudf.Series(
+        [
+            "the quick fox jumped over the lazy dog",
+            "the siamésé cat jumped under the sofa",
+            None,
+            "",
+        ]
+    )
+
+    expected = cudf.Series(
+        [
+            "the",
+            "quick",
+            "fox",
+            "jumped",
+            "over",
+            "the",
+            "lazy",
+            "dog",
+            "the",
+            "siamésé",
+            "cat",
+            "jumped",
+            "under",
+            "the",
+            "sofa",
+        ]
+    )
+
+    actual = strings.str.tokenize()
+
+    assert_series_equal(expected.to_pandas(), actual.to_pandas())
+
+
+@pytest.mark.parametrize(
+    "delimiter, expected_token_counts",
+    [
+        (" ", [10, 9, 0, 0, 1]),  # TODO: verify last count should be 1, not 5
```

Comment:
@kkraus14 @davidwendt

Counting tokens for the following string indicates only a single token, but legacy nvtext test indicates it should be 5.

```
"test_str\x01test_str\x02test_str\x03test_str\x04test_str\x05",
```

Has behavior changed, or am I using the wrong delimiter argument? See existing tests:
https://github.com/rapidsai/cudf/blob/61dacf9bb4814e0388b5e2ac99fc294646f6ef70/python/nvstrings/tests/test_text.py#L42